RESTserver/src/main/java/com/softserve/if072/restservice/service/ProductService.java

```diff
@@ -20,121 +20,151 @@
  *
  * @author Vitaliy Malisevych
  */
+
 @Service
 public class ProductService {
-    public static final Logger LOGGER = LogManager.getLogger(ProductService.class);
+
+    public static final Logger LOGGER =  LogManager.getLogger(ProductService.class);
+
     private ProductDAO productDAO;
+
     @Value("${product.notFound}")
     private String productNotFound;
 
     @Autowired
-    public ProductService(ProductDAO productDAO) {
+    public ProductService (ProductDAO productDAO) {
         this.productDAO = productDAO;
     }
 
-    public List<Product> getAllProducts(int userId) throws DataNotFoundException {
+    public List<Product> getAllProducts(int userId) {
         List<Product> getProducts = productDAO.getAllByUserId(userId);
-        List<Product> products = new ArrayList<Product>();
-        if (!getProducts.isEmpty()) {
-            for (Product getProduct : getProducts) {
-                if (getProduct.isEnabled()) {
-                    products.add(getProduct);
-                }
+        List<Product> products = new ArrayList<>();
+        if (!getProducts.isEmpty()){
+            for(Product getProduct : getProducts) {
+                if(getProduct.isEnabled()) {products.add(getProduct);}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)